src/routing/types.ts

```diff
@@ -0,0 +1,16 @@
+export enum ROUTES {
+  Alerts = 'alerts',
+  CheckDashboard = 'checks/:id',
+  Checks = 'checks',
+  ChooseCheckGroup = 'checks/choose-type',
+  Config = 'config', // config (index)
+  EditCheck = 'checks/:id/edit',
+  ViewProbe = 'probes/:id',
+  EditProbe = 'probes/:id/edit',
+  Home = 'home',
+  NewCheck = 'checks/new',
+  NewProbe = 'probes/new',
+  Probes = 'probes',
+  Redirect = 'redirect',
+  Scene = 'scene',
+}
```

Comment:
❤️

Reply:
When are we going to address the naming of enums 😁
This enum at least uses PascalCase for keys.

```javascript
// Bad 
enum AM_I_A_CONST_OR_A_ENUM {
  FOO,
  BAR,
}

// Bad 🤨
enum AM_I_A_CONST_OR_A_ENUM {
  Foo,
  Bar,
}

// Good 💖
enum ClearlyAEnum {
  Foo,
  Bar,
}
```

Reply:
https://github.com/grafana/synthetic-monitoring-app/issues/1008 -- soon, made a ticket so we don't forget :)